inference-engine/tests/functional/plugin/shared/src/behavior/layout.cpp

```diff
@@ -81,13 +88,24 @@ TEST_P(LayoutTest, NetWithLayout) {
     InferenceEngine::CNNNetwork cnnNet(function);
     if (checkLayout(layout, inputShapes)) {
         ASSERT_NO_THROW(cnnNet.getInputsInfo().begin()->second->setLayout(layout));
+
+        if (targetDevice == CommonTestUtils::DEVICE_GNA) {
+            return;
+        }
```

Comment:
in this test case ` SKIP_IF_CURRENT_TEST_IS_DISABLED();` is used. So, can we just disable this test for GNA in https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/gna/shared_tests_instances/skip_tests_config.cpp instead of hacks inside the tests itself?